### PR TITLE
COMMERCE 2651 1

### PR DESCRIPTION
**Pull request by ivicac**

### CODE REVIEW

**Comment by ivicac** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-payment-method-remote/src/main/java/com/liferay/commerce/payment/method/remote/internal/portlet/action/EditRemoteCommercePaymentMethodConfigurationMVCActionCommand.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Ivica Cardic
+ */
+@Component(
+	enabled = false, immediate = true,
+	property = {
+		"javax.portlet.name=" + CPPortletKeys.COMMERCE_PAYMENT_METHODS,
+		"mvc.command.name=editRemoteCommercePaymentMethodConfiguration"
```
Comment (dsitu) [inline modules/apps/commerce/commerce-payment-method-remote/src/main/java/com/liferay/commerce/payment/method/remote/internal/portlet/action/EditRemoteCommercePaymentMethodConfigurationMVCActionCommand.java:45]: https://github.com/liferay/liferay-portal/blob/master/modules/util/source-formatter/src/main/resources/documentation/checks/mvc_command_name_check.markdown

```suggestion
		"mvc.command.name=/commerce_payment/edit_remote_commerce_payment_method_configuration"
```

File: modules/apps/commerce/commerce-payment-method-remote/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -0,0 +1,18 @@
+activate-recurring-payment-endpoint-url=Activate Recurring Payment Endpoint URL
+authorize-payment-endpoint-url=Authorize Payment Endpoint URL
+cancel-payment-endpoint-url=Cancel Payment Endpoint URL
+cancel-recurring-payment-endpoint-url=Cancel Recurring Payment Endpoint URL
+capture-payment-endpoint-url=Capture Payment Endpoint URL
+complete-payment-endpoint-url=Complete Payment Endpoint URL
+complete-recurring-payment-endpoint-url=Complete Recurring Endpoint URL
```
Comment (dsitu) [inline modules/apps/commerce/commerce-payment-method-remote/src/main/resources/content/Language.properties:7]: Match key.
```suggestion
complete-recurring-payment-endpoint-url=Complete Recurring Payment Endpoint URL
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by ivicac** [comment]
ci:test:sf

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:forward

**Comment by marco-leo** [comment]
@riccardo-alberti FYI

**Pull request closed without merging**